### PR TITLE
Icons with moar pixels

### PR DESCRIPTION
**Pull request by seejdev**
Two changes:

- I found that when imagemagick converted the SVGs to .icos, we ended up with white squares particularly for dark mode icons. Changing the generator to convert from PNGs to .icos preserves the transparency
- Added some new icons, and now all icons are 256x256

### CODE REVIEW

**Review by James-Pickett — DISMISSED** [review]

File: ee/ui/assets/assets.go [hunk 1]
```diff
@@ -19,9 +19,21 @@ var MenubarCircleXLightmodeIco []byte
...
+var MenubarCircleXMonochromeDarkmodePng []byte
+
 //go:embed menubar-circle-x-monochrome.ico
 var MenubarCircleXMonochromeIco []byte
 
+//go:embed menubar-circle-x-monochrome-lightmode.ico
+var MenubarCircleXMonochromeLightmodeIco []byte
+
+//go:embed menubar-circle-x-monochrome-lightmode.png
+var MenubarCircleXMonochromeLightmodePng []byte
```
Comment (directionless) [inline ee/ui/assets/assets.go:35]: I'm a little confused by this. I thought we didn't need the dark/light monochrome versions. And if we do have them, we do we still have `menubar-circle-x-monochrome` ?

**Review by directionless — COMMENTED** [review]
Can we ditch the svgs?

File: ee/ui/assets/assets.go (hunk 1 above)
Comment (seejdev) [reply ee/ui/assets/assets.go:35]: Hmm I'm not entirely sure, I just dropped the new set of icons in and hit generate. Maybe Fritz renamed some?

Comment (seejdev) [reply ee/ui/assets/assets.go:35]: To better answer your question - yes, for macOS we would only need one monochrome version which can be used as a template image for both dark and light modes. Not sure if Fritz had anything else in mind.

I can remove the `-monochrome-dark/light` ones and leave just the `-monochrome` ones if we don't see any need for them right now.

Comment (directionless) [reply ee/ui/assets/assets.go:35]: I'd bet Fritz gave us more than we need

Comment (directionless) [reply ee/ui/assets/assets.go:35]: (mostly I'm trying minimize the size we use)

Comment (seejdev) [reply ee/ui/assets/assets.go]: Removed  the `-monochrome-dark/light` ones, regenerated.

**Review by RebeccaMahany — APPROVED** [review]

**Pull request merged**